src/SOS/Strike/sosdocs.txt

```diff
@@ -682,10 +682,16 @@ The arguments in detail:
 
 -allReady Specifying this argument will allow for the display of all objects 
           that are ready for finalization, whether they are already marked by 
-          the GC as such, or whether the next GC will.  The objects that are 
-          not in the "Ready for finalization" list are finalizable objects that 
-          are no longer rooted.  This option can be very expensive, as it 
-          verifies whether all the objects in the finalizable queues are still 
+          the GC as such or not. The former means GC already put them in the 
```

Comment:
What about Strike/sosdocsunix.txt (linux/macos help)?

Reply:
ahh yes, also updated that one.